README.md

```diff
@@ -36,6 +36,20 @@ In your feature:
   Then I save screenshot
 ```
 
+test feature:
```

Comment:
please remove this  - we do not need this in readme - it is a test for this package